vignettes/customizing-module-output.Rmd

```diff
@@ -0,0 +1,327 @@
+---
+title: "Customizing Module Output"
+author: "NEST CoreDev"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Customizing Module Output}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis.
```

Comment:
I'd skip this line at all. This is decorator vignette and everyone should already know what is teal. Also "powerful" is a bragging which shouldn't really have a place in the vignette. (this is just my opinion)

```suggestion
```

Reply:
Agree.